rune/libenclave/internal/runtime/pal/skeleton/README.md

```diff
@@ -227,7 +227,11 @@ The options provided by `sgxsign` tool are as followed:
 | -p, --product                       | Generate production enclave. Debug enclave is generated by default.                                    |
 | -s, --mmap-size value               | Launch an enclave with memory size equals to value(bytes).                                             |
 | -a, --attrs value                   | Launch an enclave with attributes equals to value(hex).                                                |
+| -A, --attrs-mask value              | Enforce the attributes specified by --attrs.
+                                   |
 | -x, --xfrm value                    | Launch an enclave with xfrm equals to value(hex).                                                      |
+| -X, --xfrm-mask value               | Enforce the xfrm specified by --xfrm.
```

Comment:
It is better to show the unit of `--xfrm-mask`, such as `hex`
Same to `--attrs-mask`

Reply:
Done.